### PR TITLE
docs: fix typos in README.md files

### PR DESCRIPTION
**Pull request by Emmanuel-Melon**
## Description

There were typos in the README.md files of the following directories:
- tools/hawk
- tools/spanner
- tools/user_migration

## Testing

Reviewing the README.md files.

## Issue(s)

Closes [529](https://github.com/mozilla-services/syncstorage-rs/issues/529).

### CODE REVIEW

**Comment by Emmanuel-Melon** [comment]
Accidentally added commits from the previous PR here as well.

**Comment by fzzzy** [comment]
To fix it, `git reset --hard [the commit hash at the head of master]` and then `git cherry-pick 
1981d9f`

Finally, `git push -f`

**Comment by Emmanuel-Melon** [comment]
> To fix it, `git reset --hard [the commit hash at the head of master]` and then `git cherry-pick 1981d9f`
> 
> Finally, `git push -f`

Thank you but I just did this and it seems like the commit got removed from my clone repo but not the upstream repo.

**Comment by tublitzed** [comment]
@Emmanuel-Melon just to confirm, have you configured your system to point to this repo [as a remote](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) for your fork?

If so, can you then:

1. `git fetch upstream`
2. `git merge upstream/master`
3. Then merge your local master into your local branch and you should be able to alternatively try using [rebase](https://git-scm.com/docs/git-rebase) if the cherry-picking suggestion above doesn't work.

**Comment by Emmanuel-Melon** [comment]
> @Emmanuel-Melon just to confirm, have you configured your system to point to this repo [as a remote](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) for your fork?
> 
> If so, can you then:
> 
> 1. `git fetch upstream`
> 2. `git merge upstream/master`
> 3. Then merge your local master into your local branch and you should be able to alternatively try using [rebase](https://git-scm.com/docs/git-rebase) if the cherry-picking suggestion above doesn't work.

Hey, there!
Yes, that's exactly what I did and I had tried the cherry picking suggestion which seemed to work but with my forked origin instead of upstream.
Let me try again and publish the PR when it works.

**Comment by Emmanuel-Melon** [comment]
> > @Emmanuel-Melon just to confirm, have you configured your system to point to this repo [as a remote](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) for your fork?
> > If so, can you then:
> > 
> > 1. `git fetch upstream`
> > 2. `git merge upstream/master`
> > 3. Then merge your local master into your local branch and you should be able to alternatively try using [rebase](https://git-scm.com/docs/git-rebase) if the cherry-picking suggestion above doesn't work.
> 
> Hey, there!
> Yes, that's exactly what I did and I had tried the cherry picking suggestion which seemed to work but with my forked origin instead of upstream.
> Let me try again and publish the PR when it works.

Deleted my local branch, checked out the branch again then cherry picked and it worked.

**Comment by Emmanuel-Melon** [comment]
@tublitzed Commit signed.
